jQuery/package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "dx-jquery",
+  "version": "1.0.0",
+  "type": "module",
+  "scripts": {
+    "start": "npm run lite",
+    "lite": "lite-server -c ./bs-config.json"
+  },
+  "devDependencies": {
+    "eslint": "^4.18.2",
+    "htmlhint": "^0.11.0",
+    "lite-server": "^2.5.4"
+  },
+  "dependencies": {
+    "devextreme": "^22.1.5",
+    "jquery": "^3.5.1"
+  }
```

Let's update the ```devextreme``` package version and add ```rrule``` to dependencies:

```suggestion
  "dependencies": {
    "devextreme": "^22.2.4",
    "jquery": "^3.5.1",
    "rrule": "^2.7.1"
  }
```

And regenerate the package-lock.json